search/domain/service.go

```diff
@@ -64,10 +64,10 @@ type (
 		SelectedDesc bool
 		// Asc - represents the field that is used to trigger ascending search.
 		// Deprecated: use "Field" and "SelectedAsc" instead to set which field should be sortable
-		Asc string
+		Asc string // Should it be deprecated ?? marked as deprecated in 2019 and used in 2020
```

Comment:
no hard opinion on that, maybe we have a chat with more contributors on this.

Reply:
since nobody seems to care feel free to remove your comment and the deprecation notice.